src/checkout/component.js

```diff
@@ -65,6 +65,10 @@ export let Checkout : Component<CheckoutPropsType> = create({
                 throw new Error(`Expected payment id or token to be passed, got ${ token }`);
             }
 
+            if (isPayPalDomain() && window.xprops && window.xprops.checkoutUri) {
+                return window.xprops.checkoutUri;
```

Comment:
Should we also be pre-pending the `config.checkoutUrls[props.env]` here?

Reply:
Hm... so my understanding is that `xprops` is passed from the button component props, so `window.xprops.checkoutUri` is the already built URL. Are there cases where that's not true?